lib/runner.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Runner do
   def start do
-    Display.print_board([0, 1, 2, 3, 4, 5, 6, 7, 8])
+    Console.print_welcome_msg
+    Console.print_board([0, 1, 2, 3, 4, 5, 6, 7, 8])
```

Comment:
I would recommend clarifying this by creating an initial board variable, or something like that.